javascript/apps/taiga/src/app/modules/feature-user-settings/components/deleting-account-modal/deleting-account-modal.component.html

```diff
@@ -0,0 +1,184 @@
+<!--
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+Copyright (c) 2023-present Kaleidos INC
+-->
+<ng-container *transloco="let t">
+  <ng-container *ngIf="model$ | async as vm">
+    <tg-ui-modal
+      class="confirm-close"
+      [open]="open"
+      [width]="500"
+      (requestClose)="cancel.next()">
+      <div
+        data-test="deleting-account-modal"
+        role="dialog"
+        aria-labelledby="deleting-account"
+        aria-modal="true">
+        <h1
+          class="title"
+          id="deleting-account">
+          {{ t('user_settings.account.delete.confirmation_modal.title') }}
+        </h1>
+        <h2 class="visually-hidden">
+          {{ t('user_settings.account.delete.confirmation_modal.description') }}
+        </h2>
+
+        <tui-scrollbar>
+          <ng-container *ngIf="vm.accountInfo?.workspaces as workspaces">
+            <section
+              class="user-admin-list"
+              *ngIf="workspaces.length">
+              <tg-ui-inline-notification status="warning">
+                <p>
+                  {{
+                    t('user_settings.account.delete.confirmation_modal.caution')
+                  }}
+                </p>
+                <p
+                  subtitle
+                  [innerHTML]="
+                    t(
+                      'user_settings.account.delete.confirmation_modal.only_admin_workspaces',
+                      {
+                        workspaces: workspaces.length
+                      }
+                    ) | safeHtml
+                  "></p>
+              </tg-ui-inline-notification>
+              <p class="description">
+                {{
+                  t(
+                    'user_settings.account.delete.confirmation_modal.workspace_description'
+                  )
+                }}
+              </p>
+              <h3
+                class="visually-hidden"
+                *ngIf="workspaces.length > 1">
+                {{
+                  t(
+                    'user_settings.account.delete.confirmation_modal.workspace_a11y_title'
+                  )
+                }}
+              </h3>
+
+              <div class="workspaces">
+                <div
+                  class="workspace"
+                  data-test="delete-user-workspace"
+                  *ngFor="let workspace of workspaces; trackBy: trackById">
+                  <tg-ui-avatar
+                    class="workspace-avatar"
+                    [color]="workspace.color"
+                    type="dark"
+                    size="m"
+                    [name]="workspace.name">
+                  </tg-ui-avatar>
+
+                  <span class="workspace-name">{{ workspace.name }}</span>
+
+                  <tg-projects-dropdown
```

Comment:
When the user do focus the side border is not visible. 
![image](https://github.com/taigaio/taiga/assets/5970959/2d1d9cd0-06e5-4887-8789-f8db6fe4725a)